### DISCUSSION TITLE
Play All

### DISCUSSION THREAD

**Discussion by TJoeT** [music-assistant/support#924] [Feature Polls]
Is there a hassle free " Play All " from a local Directory button ??

Select All to add to the queue don't see that.
Select All to add to a playlist don't see that.

Any change you could make a youtube video showing how you see this being used correctly?

**Comment by OzGav** [comment]
There is an issue related to this https://github.com/music-assistant/hass-music-assistant/issues/821

However, the process is go to one of ARTISTS/ALBUMS/TRACKS view. Press CTRL-A and down the bottom will appear a button marked ACTIONS. Select that and then PLAY NOW
